docs: step inspector article is updated (#132)
Innoactive/Creator: .Documentation/articles/innoactive-creator/step-inspector.md

Learn more about the [default conditions](default-conditions.md).

[![Add a condition to a step](../images/step-inspector/add-condition.gif "How to add a condition to a step.")](../images/step-inspector/add-condition.gif)

## Reorder Behaviors, Transitions, and Conditions

To reorder a behavior, transition, or condition, click on the <img src="../images/step-inspector/icon_arrow_down_dark.png" alt="Arrow Down Icon" height="15px"/> button to move it down or on the <img src="../images/step-inspector/icon_arrow_up_dark.png" alt="Arrow Up Icon" height="15px"/> button to move it up.

Tip: You can change the priority of transitions by sorting them from top to bottom. If two or more transitions are completed at the same time, the highest one in the list will be prioritized.

[![Reorder transitions](../images/step-inspector/reorder-entities.gif "How to reorder behaviors, transitions, and conditions.")](../images/step-inspector/reorder-entities.gif)

## Unlock Objects in Current Step

> By default, every property (Touching, Grabbing, Using, ...) on an training scene object that is not needed by a condition is locked (non-interactable) during a step. In the `Unlocked Objects` tab you can see which object properties are active and which are not. You can also manually drag in other training scene objects of the scene to manually unlock its properties per step.

Author: ```suggestion
By default, every property (Touching, Grabbing, Using, ...) on a training scene object that is not needed by a condition is locked (non-interactable) during a step. In the `Unlocked Objects` tab you can see which object properties are active and which are not. You can also manually drag in other training scene objects from the scene to manually unlock their properties per step.
```
